server/postgres/db_setup_draft.sql

```diff
@@ -335,11 +335,11 @@ CREATE TABLE zinvites (
 CREATE INDEX zinvites_zid_idx ON zinvites USING btree (zid);
 
 -- TODO flush regularly
-CREATE TABLE password_reset_tokens (
```

Comment:
Could this change have an impact on production? I wonder if there needs to be a migration script to change the table name on an existing table.